examples/Order/Capture.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Examples\Order\Capture;
```

Comment:
O namespace vai até o nome da pasta, não precisa colocar o nome do arquivo.
```suggestion
namespace Examples\Order;
```